alma/lib/Utils/functions.php

```diff
@@ -150,15 +150,26 @@ function almaFormatPrice($cents, $id_currency = null)
 function getDateFormat($locale, $timestamp)
 {
     try {
-        $formatter = new IntlDateFormatter($locale, IntlDateFormatter::MEDIUM, IntlDateFormatter::NONE);
-        if ($formatter === null) {
-            throw new IntlException(intl_get_error_message());
+        if (class_exists(IntlDateFormatter::class)) {
+            $formatter = new IntlDateFormatter($locale, IntlDateFormatter::MEDIUM, IntlDateFormatter::NONE);
+            return $formatter->format($timestamp);
         }
+    } catch (Exception $e) {}
```

Comment:
```suggestion
    } catch (Exception $e) {
        // We don't need to deal with this Exception because a fallback exists in default return statement
    }
```

Reply:
@Benjamin-Freoua-Alma according this [sonar issue](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=102&id=alma_alma-installments-prestashop&open=AYCKr5CYjqOfzcBvLhLy), we should not let empty block of code (`{}`).

> Most of the time a block of code is empty when a piece of code is really missing. So such empty block must be either filled or removed.
>
> When a block contains a comment, this block is not considered to be empty.